examples/common/tracing/TraceHandlers.h

```diff
@@ -53,6 +54,8 @@ class TraceStreamLog : public TraceStream
 
     void FinishEvent() override { ChipLogAutomation("TraceStream:%s", mStreamLine.c_str()); }
 
+    void CleanJsonBuf(void) override {}
```

Comment:
use `()` instead of `(void)` throughout.

Reply:
Fixed